frontend/src/helpers/utils.js

```diff
@@ -0,0 +1,3 @@
+export function ipToLong(ip) {
+  return ip.split('.').reduce((acc, octet) => (acc << 8) + parseInt(octet, 10), 0);
```

Comment:
what about IPv6?

Reply:
While IPv4 addresses are available by default, and both addresses are showed in one column, I think that sorting on IPv4 is enough.

Reply:
And what if a user only uses IPv6? parseInt might return NaN in that case

Reply:
maybe we can add a check for IPv6 (string contains ':'), just to make sure that the code always produces valid resutls

Reply:
You're right, I missed this. Will update

Reply:
Fixed
![image](https://github.com/user-attachments/assets/12f6faad-653f-4e8f-8c6a-1734c6c194c4)